app/src/androidTest/java/com/radixdlt/android/apps/wallet/WorkaroundIntentsTestRule.kt

```diff
@@ -0,0 +1,57 @@
+/*
+ * Copyright (C) 2015 The Android Open Source Project
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
```

Comment:
All other repos are using MIT, which I think is the company wide standard? We should consolidate this :)